data/assets/nightsky/galactic_band.asset

```diff
@@ -28,53 +28,56 @@ local GalacticLine = {
 }
 
 
-local ShowGalacticBand = {
-  Identifier = "os.nightsky.ShowGalacticBand",
-  Name = "Show galactic band",
+local ShowGalacticEquator = {
+  Identifier = "os.nightsky.ShowGalacticEquator",
+  Name = "Show galactic equator",
   Command = [[
-    openspace.fadeIn("Scene.GalacticBand.Renderable")
+    openspace.fadeIn("Scene.GalacticLine.Renderable")
```

Comment:
When renaming this to Equator, should the scenegraphnode be renamed the same way?